pkg/binding/example_implementing_test.go

```diff
@@ -43,7 +44,7 @@ type ExSender struct {
 	transformers binding.TransformerFactories
 }
 
-func NewExSender(w io.Writer, factories ...binding.TransformerFactory) binding.Sender {
+func NewExSender(w io.Writer, factories ...binding.TransformerFactory) bindings.Sender {
```

Comment:
I think we should get rid of those tests or refactor them somehow, they are meant to explain to bindings implementers how to create a new binding. But I think it's more self explicative the http implementation honestly, even if it's a bit more complex...

Reply:
In the next PR I moved them